kotlinx.fuzz.api/src/main/kotlin/kotlinx/fuzz/KFuzzer.kt

```diff
@@ -246,20 +246,20 @@ interface KFuzzer {
      * insufficiently long. In this case it will be shorter.
      *
      * @param maxLength the maximum length of the array
-     * @param range denotes desired range for each value. Should be a subset of [Float.MIN_VALUE, Float.MAX_VALUE].
-     * [Float.MIN_VALUE, Float.MAX_VALUE] by default.
+     * @param range denotes desired range for each value. Should be a subset of [-Float.MAX_VALUE, Float.MAX_VALUE].
+     * [-Float.MAX_VALUE, Float.MAX_VALUE] by default.
```

Comment:
Can you comment on this change, please?

Reply:
It was my mistake in the first place. For floats and doubles `MIN_VALUE` is minimal absolute value, but in order to get the least possible number you need to have `-MAX_VALUE`